src/sui/themes/rtd-site/templates/footer.overrides

```diff
@@ -0,0 +1,205 @@
+    margin: floor(@footerLinksLineHeight / 2) 0 ceil(@footerLinksLineHeight / 2);
+  }
+  p,
+  li {
+    font-size: 14px;
+  }
+  a {
+    color: @footerColor;
+  }
+  a:hover {
+    color: @footerHoverColor;
+  }
+}
+
+/*------------------------
+      connections
+-------------------------*/
+#footer section.main {
+
+  .connections {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding-bottom: @footerRowsPadding;
+    border-bottom: 1px solid @footerSeparatorsColor;
+    position: relative;
+  }
+
+}
+
+/*-------------------
+        links
+--------------------*/
+#footer section.main {
+
+  .links {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+  }
+
+  .links ul {
+    list-style: none;
+    margin: 0;
+    padding: @footerColumnsPaddingV @footerColumnsPaddingH;
+    // flex-basis: floor(100%/(@footerLinksColumnsCount+2));
+    // flex-grow: 1;
+    // flex-shrink: 0;
+  }
+
+  .links a {
+    text-decoration: none;
+  }
+  .links a:hover {
+    text-decoration: underline;
+  }
+}
+
+/*-------------------
+      Marketing
+--------------------*/
+#footer section.main {
+
+  .marketing {
+    flex: 1 1 400px;
+  }
+
+}
+
+/*------------
+  Newsletter
+-------------*/
+#footer section.main {
+
+  .newsletter {
+    padding: @footerColumnsPaddingV @footerColumnsPaddingH;
+  }
+
+}
+
+/*------------
+    social
+-------------*/
+#footer section.main {
+
+  .social {
+    padding: @footerColumnsPaddingV @footerColumnsPaddingH;
+    font-size: 22px;
+  }
+
+}
+
+/*-------------------
+        logo
+--------------------*/
+.logo {
+  background-color: @footerBackground;
+  padding: 10px;
+  width: @footerLogoWidth;
+  position: absolute;
+  bottom: -(@footerLogoWidth / 2);
+  left: 50%;
+  margin-left: -(@footerLogoWidth / 2);
+
+  img {
+    opacity: .18;
+  }
+}
+
+/*-----------------------
+         sponsors
+-------------------------*/
+#footer section.main {
+
+  .sponsors {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    align-content: flex-start;
+
+    max-width: 700px;
+    margin: 0 auto;
+    padding-top: @footerRowsPadding;
+  }
+  .sponsor {
+    padding: @footerColumnsPaddingV @footerColumnsPaddingH;
+    flex: 0 1 (100% / 6);
+    opacity: .3;
+  }
+  .sponsor:hover {
+    opacity: .5;
+  }
+
+}
+
+
+/*-----------------------------
+          Copyrights
+------------------------------*/
+#footer section.copyrights {
+  color: @footerCopyrightsColor;
+  padding: 20px 0;
+  font-size: 90%;
+
+  a {
+    color: @footerCopyrightsColor;
+  }
+}
+
+#footer section.copyrights .container {
+  display: flex;
+  justify-content: space-between;
+}
+
+/*------------
+     left
+-------------*/
+#footer section.copyrights {
+
+  .left {}
+
+}
+
+/*------------
+     right
+-------------*/
+#footer section.copyrights {
+
+  .right {
+
+    a {
+      text-decoration: none;
+    }
+  }
+
+  span.lang {}
+  span.version {}
+
+}
```

Comment:
I raised questions on why these sort of rules are needed, as opposed to accomplishing this with SUI grid/etc, but noting here that a lot of these rules look redundant to SUI components and could mostly go away. I did end up with a few rules when implementing the footer in the application, perhaps it's better to work from that as a base.

Reply:
Solved. The footer was remade from the forked app footer. I also looked for other unused rules to remove.